dockstore-client/src/main/java/io/dockstore/client/cli/Client.java

```diff
@@ -345,16 +346,29 @@ private static void cwl(List<String> args) {
             kill("Please provide a container.");
         }
 
-        String path = args.get(0);
+        String[] parts = args.get(0).split(":");
+
+        String path = parts[0];
+
+        String tag = (parts.length > 1) ? parts[1] : null;
 
+        out("TAG: " + tag);
         try {
             Container container = containersApi.getContainerByPath(path);
             if (container.getHasCollab()) {
-                FileResponse collab = containersApi.cwl(container.getId());
-                if (collab.getContent() != null && !collab.getContent().isEmpty()) {
-                    out(collab.getContent());
-                } else {
-                    out("No cwl file found.");
```

Comment:
In all of these error conditions, the method should return immediately (for clarity) and the CLI should return a non-zero exit code.